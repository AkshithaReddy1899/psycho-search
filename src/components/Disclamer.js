/* eslint-disable max-len */
import React from 'react';

function Disclamer() {
  return (
    <div className="m-6 p-4 w-1/2 mx-auto font-serif leading-8">
      <h2 className="text-2xl text-red-500">Warning</h2>
      <div className="pt-6">
        Hello, I hope you are doing well.
        <br />
        <p className="text-left text-2xl leading-10">
          All the information presented here is from the internet, which might or might not be true. My intention behind this web application is purely educational. However, you can always mail me at
          <a href="mailto:akshithareddy1899@gmail.com" className="text-green-600 decoration-solid" onClick={() => navigator.clipboard.writeText('akshithareddy1899@gmail.com')}> akshithareddy1899@gmail.com </a>
          <br />
          <ul className="list-disc text-lg leading-8 p-8">
            <li>If there is any information that bothers you</li>
            <li>If you feel any information is incorrectly presented</li>
            <li>You have an idea that could make this application even better</li>
            <li>If you just want to let me know your thoughts on this</li>
          </ul>
          I would be pleased to hear more and correct any information or add more features to this web application that you think would be good.
        </p>
        <br />
        <p className="text-left text-lg">
          Best,
          <br />
          Akshitha Reddy.
        </p>
      </div>
    </div>
  );
}

export default Disclamer;
