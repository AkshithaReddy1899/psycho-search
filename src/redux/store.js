import {
  legacy_createStore as createStore,
  combineReducers,
  applyMiddleware,
  compose,
} from 'redux';
import thunk from 'redux-thunk';
import logger from 'redux-logger';
import personsReducer from './person';

const rootReducer = combineReducers({
  person: personsReducer,
});

const store = createStore(rootReducer, compose(applyMiddleware(logger, thunk)));

export default store;
