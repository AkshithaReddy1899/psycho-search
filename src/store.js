import { configureStore } from '@reduxjs/toolkit';
import personSlice from './slices/personSlice';

const reducer = {
  person: personSlice,
};

const store = configureStore({
  reducer,
  devTools: true,
  middleware: (getDefaultMiddleware) => getDefaultMiddleware({
    serializableCheck: false,
  }),
});

export default store;
