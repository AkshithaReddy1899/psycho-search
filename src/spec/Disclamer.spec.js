import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import matches from './test-utils';
import store from '../redux/store';
import Disclamer from '../components/Disclamer';

describe('Home', () => {
  it('Home matched snapshot', () => {
    matches(<Provider store={store}><BrowserRouter><Disclamer /></BrowserRouter></Provider>);
  });
});
