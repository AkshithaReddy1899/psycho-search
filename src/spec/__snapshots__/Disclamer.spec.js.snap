// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Home matched snapshot 1`] = `
<div
  className="m-6 p-4 w-1/2 mx-auto font-serif leading-8"
>
  <h2
    className="text-2xl text-red-500"
  >
    Warning
  </h2>
  <div
    className="pt-6"
  >
    <p
      className="text-left text-2xl leading-10"
    >
      All the information presented here is from the internet, which might or might not be true. My intention behind this web application is purely educational. Also, these are not the only psychopaths in the world. I will be adding more data in the near future. However, you can always e-mail me at
      <a
        className="text-green-600 decoration-solid"
        href="mailto:akshithareddy1899@gmail.com"
        onClick={[Function]}
      >
         akshithareddy1899@gmail.com 
      </a>
      <br />
      <ul
        className="list-disc text-lg leading-8 p-8"
      >
        <li>
          If there is any information that bothers you
        </li>
        <li>
          If you feel any information is incorrectly presented
        </li>
        <li>
          You have an idea that could make this application even better
        </li>
        <li>
          If you just want to let me know your thoughts on this
        </li>
      </ul>
      I would be pleased to hear more and correct any information or add more features to this web application that you think would be good.
    </p>
    <br />
    <p
      className="text-left text-lg"
    >
      Best,
      <br />
      Akshitha Reddy.
    </p>
  </div>
</div>
`;
